changelog/unreleased/9425

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix Account Filter for Server Activity tab
+
+This fixes the "bug" part of the reported issue. The UI/UX improvements
+are done separately.
```

Comment:
I'm not sure the second sentence should end up in the change of.

Reply:
I'd also rather add PRs that implement UI changes to the existing changelog item instead of announcing them in this one.